New package: HAPIClient v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: HAPIClient
- Repository: https://github.com/Beforerr/HAPIClient.jl
- Created by: @Beforerr
- Version: v0.1.0
- Commit: fcd3964d484e085a065dc2ce396bb07529de4c73
- Reviewed by: @Beforerr
- Reference: https://github.com/Beforerr/HAPIClient.jl/issues/1#issuecomment-2758667234
- Description: A Julia client for the Heliophysics Application Programmer's Interface (HAPI)
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359807a4391e9e22437aca15cc8cee6488300cd13e605e7941ee742b79c84a88c7ca3e6d17c9e95e6c5a74352b65635b8a3730e256406c47df21d9ca805b0a6ab1aabccc3e797ba1ad2bc4dc7390074cb5c88e482f3adfb9c9039715db07b37d2085109896232ab7155bdf5f3a5ab721d4b1d9a269adcd79edb50ed6a4205f8b87f7 -->

goerz commented:
Thank you for submitting your package! Could you add a little bit of documentation before registering? [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README. An important part of packages in General is that any potential user can figure out what the package is about and how to get started with using it. That is really difficult when there is no documentation.

In the longer term, I definitely recommend setting up a [Documenter](https://documenter.juliadocs.org/stable/)-based documentation. Before a `v1.0` release, or for smaller packages that can be effectively described entirely with their `README`, that's not a requirement, though.

JuliaRegistrator committed:
New package: HAPIClient v0.1.0

UUID: 0ce0b37c-8851-4c5b-98a3-e691a5f19c53
Repo: https://github.com/Beforerr/HAPIClient.jl.git
Tree: c68781e9f4f1573cd132a5453bbe3f6974d8401f

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

Beforerr commented:
[noblock]
@goerz  Thanks for the feedback! I’ve updated the docs and added a description along with a basic usage example to the README. Let me know if anything else is needed.

goerz commented:
[noblock]

Thanks! Seems good enough for a 0.1 release…
